tests/dream/geant4_reduction_test.py

```diff
@@ -188,3 +182,14 @@ def _assert_contains_source_info(cif_content: str) -> None:
 def _assert_contains_author_info(cif_content: str) -> None:
     assert "audit_contact_author.name 'Jane Doe'" in cif_content
     assert 'audit_contact_author.email jane.doe@ess.eu' in cif_content
```

Comment:
Orcid ID could be tested as well.

Reply:
I didn't want to replicate all tests from ScippNeutron. But I can add this.